packages/design-picker/src/components/generated-design-picker.tsx

```diff
@@ -18,19 +18,22 @@ interface GeneratedDesignThumbnailProps {
 	thumbnailUrl: string;
 	isSelected: boolean;
 	onPreview: () => void;
+	index: number;
 }
 
 const GeneratedDesignThumbnail: React.FC< GeneratedDesignThumbnailProps > = ( {
 	slug,
 	thumbnailUrl,
 	isSelected,
 	onPreview,
+	index,
```

Comment:
I prefer to use `label` as `prop` to let its parent decide how to call it. Besides, we also need to translate aria-label

Reply:
Thanks @arthur791004 👍  I have pushed the changes

Reply:
I think @arthur791004 is saying that we should pass the entire label string as a prop instead.
For instance `label={ sprintf( __( 'Preview design option %s' ), index )  }`